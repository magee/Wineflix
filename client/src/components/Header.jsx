import React from 'react';
import { Navbar, Nav, NavItem } from 'react-bootstrap';

//const logo = require ('../assets/classygazelles_small.png');

const Header = (props) => {
  return (
    <Navbar>
     <Navbar.Header>
          <Navbar.Brand>
            <a className="site-logo" href="/">Wineflix!</a>
          </Navbar.Brand>
        </Navbar.Header>
        <Navbar.Collapse>
          <Nav pullRight>
            <NavItem
              eventKey={1}
              href="signup">
              Create account
            </NavItem>
            <NavItem
              eventKey={2}
              href="login">
              Login
            </NavItem>
          </Nav>
        </Navbar.Collapse>
    </Navbar>
    )
  }

export default Header;


