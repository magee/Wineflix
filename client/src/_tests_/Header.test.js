import React from 'react';
import { shallow } from 'enzyme';
import Header from '../components/Header.jsx';

describe('Header', () => {
  const mockFn = jest.fn();
  it('should be defined', () => {
    expect(Header).toBeDefined();
  });
  it('should render correctly', () => {
    const tree = shallow(
      <Header class="header" />
    );
    expect(tree).toMatchSnapshot();
  });
});