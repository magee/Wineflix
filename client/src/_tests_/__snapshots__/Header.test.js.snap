// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    class="header"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "bsStyle": "default",
      "children": Array [
        <NavbarHeader>
          <NavbarBrand>
            <a
              className="site-logo"
              href="/"
            >
              Wineflix!
            </a>
          </NavbarBrand>
        </NavbarHeader>,
        <NavbarCollapse>
          <Nav
            bsClass="nav"
            justified={false}
            pullLeft={false}
            pullRight={true}
            stacked={false}
          >
            <NavItem
              active={false}
              disabled={false}
              eventKey={1}
              href="signup"
            >
              Create account
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={2}
              href="login"
            >
              Login
            </NavItem>
          </Nav>
        </NavbarCollapse>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <NavbarBrand>
            <a
              className="site-logo"
              href="/"
            >
              Wineflix!
            </a>
          </NavbarBrand>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              className="site-logo"
              href="/"
            >
              Wineflix!
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Wineflix!",
              "className": "site-logo",
              "href": "/",
            },
            "ref": null,
            "rendered": "Wineflix!",
            "type": "a",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Nav
            bsClass="nav"
            justified={false}
            pullLeft={false}
            pullRight={true}
            stacked={false}
          >
            <NavItem
              active={false}
              disabled={false}
              eventKey={1}
              href="signup"
            >
              Create account
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={2}
              href="login"
            >
              Login
            </NavItem>
          </Nav>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "nav",
            "children": Array [
              <NavItem
                active={false}
                disabled={false}
                eventKey={1}
                href="signup"
              >
                Create account
              </NavItem>,
              <NavItem
                active={false}
                disabled={false}
                eventKey={2}
                href="login"
              >
                Login
              </NavItem>,
            ],
            "justified": false,
            "pullLeft": false,
            "pullRight": true,
            "stacked": false,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "children": "Create account",
                "disabled": false,
                "eventKey": 1,
                "href": "signup",
              },
              "ref": null,
              "rendered": "Create account",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "children": "Login",
                "disabled": false,
                "eventKey": 2,
                "href": "login",
              },
              "ref": null,
              "rendered": "Login",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsStyle": "default",
        "children": Array [
          <NavbarHeader>
            <NavbarBrand>
              <a
                className="site-logo"
                href="/"
              >
                Wineflix!
              </a>
            </NavbarBrand>
          </NavbarHeader>,
          <NavbarCollapse>
            <Nav
              bsClass="nav"
              justified={false}
              pullLeft={false}
              pullRight={true}
              stacked={false}
            >
              <NavItem
                active={false}
                disabled={false}
                eventKey={1}
                href="signup"
              >
                Create account
              </NavItem>
              <NavItem
                active={false}
                disabled={false}
                eventKey={2}
                href="login"
              >
                Login
              </NavItem>
            </Nav>
          </NavbarCollapse>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <NavbarBrand>
              <a
                className="site-logo"
                href="/"
              >
                Wineflix!
              </a>
            </NavbarBrand>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <a
                className="site-logo"
                href="/"
              >
                Wineflix!
              </a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Wineflix!",
                "className": "site-logo",
                "href": "/",
              },
              "ref": null,
              "rendered": "Wineflix!",
              "type": "a",
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Nav
              bsClass="nav"
              justified={false}
              pullLeft={false}
              pullRight={true}
              stacked={false}
            >
              <NavItem
                active={false}
                disabled={false}
                eventKey={1}
                href="signup"
              >
                Create account
              </NavItem>
              <NavItem
                active={false}
                disabled={false}
                eventKey={2}
                href="login"
              >
                Login
              </NavItem>
            </Nav>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "nav",
              "children": Array [
                <NavItem
                  active={false}
                  disabled={false}
                  eventKey={1}
                  href="signup"
                >
                  Create account
                </NavItem>,
                <NavItem
                  active={false}
                  disabled={false}
                  eventKey={2}
                  href="login"
                >
                  Login
                </NavItem>,
              ],
              "justified": false,
              "pullLeft": false,
              "pullRight": true,
              "stacked": false,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "active": false,
                  "children": "Create account",
                  "disabled": false,
                  "eventKey": 1,
                  "href": "signup",
                },
                "ref": null,
                "rendered": "Create account",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "active": false,
                  "children": "Login",
                  "disabled": false,
                  "eventKey": 2,
                  "href": "login",
                },
                "ref": null,
                "rendered": "Login",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
